kornia/augmentation/random.py

```diff
@@ -157,36 +151,40 @@ def _get_perspective_params(batch_size: int, width: int, height: int, distortion
     fx: float = distortion_scale * width / 2
     fy: float = distortion_scale * height / 2
 
-    factor = torch.tensor([fy, fx]).view(-1, 1, 2)
-
-    rand_val: torch.Tensor = Uniform(0, 1).rsample((batch_size, 4, 2))
-    offset = 2 * factor * rand_val - 1
+    factor = torch.tensor([[
+        [fy, fx],
```

Comment:
Have you verified this? I think it's wrong, the points are returned in y,x order but get_perspective_transform expects x,y

Reply:
This naming strategy actually existed in the very beginning. Our parameter can produce the same results in TorchVision.